### PR TITLE
feat: Add TimeLine component

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: src/components/TimeLine/TimeLine.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
+.time-line {
+  overflow: hidden;
+  margin: 40px -16px 36px;
```
Comment (iceicery) [inline src/components/TimeLine/TimeLine.css:3]: Brilliant! It looks like a negative margin solve our problem.

**Review by iceicery — CHANGES_REQUESTED** [review]
Nice work! I like everything functions well and works smoothly. You applied several different ways to move the time line: slide, click time item and click the dot. That is very thoughtful and easy for users to use. Very impressive!

File: src/components/TimeLine/TimeLine.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  function handleTouchMove(e) {
+    setFinalSwipeX(e.nativeEvent.touches[0].clientX);
+  }
+
+  function moveTimeLine(ind) {
+    setCurrentRightPx(ind * 272);
+  }
+
+  function handleTouchEnd() {
+    let ind = currentTimeIndex + 1 <= 4 ? currentTimeIndex + 1 : 4;
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/components/TimeLine/TimeLine.js:25]: Thanks for spotting that!

**Review by iceicery — APPROVED** [review]
Great job! Approved.

**Pull request merged**